src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -38,8 +40,11 @@ public IntegrityCheck(BibDatabaseContext bibDatabaseContext) {
             return result;
         }
 
-        result.addAll(new AuthorNameChecker("author").check(entry));
-        result.addAll(new AuthorNameChecker("editor").check(entry));
+        for (String fieldName : entry.getFieldNames()) {
+            if (InternalBibtexFields.getFieldExtras(fieldName).contains(BibtexSingleFieldProperties.PERSON_NAMES)) {
```

Comment:
That list would typically be longer, so more iterations. Also, if we have e.g. a DateChecker, one can add a simple if-clause below this. Probably one should add the JOURNAL_NAME property to a few more fields and run that checker for those fields.